### PR TITLE
APP-27128: stack `Toasts` when multiple `Toasts` at the same time

### PR DESCRIPTION
**Pull request by habxtech**
See [APP-27128](https://habxfr.atlassian.net/browse/APP-27128): Modify toasters display when clicking on "envoyer par mail" in recap in mode withcontact.

### CODE REVIEW

File: src/notify/ToasterList.interface.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export type StateToast = {
   options: ToastOptions
   open: boolean
   id: number
-  timeout: number | null | undefined
+  timeout?: number | null | undefined
```
Comment (jean9696) [inline src/notify/ToasterList.interface.ts]: This is still required

File: src/notify/ToasterList.style.ts [hunk 2]
```diff
@@ -19,14 +20,17 @@ export const ToasterListContainer = styled.div`
...
 export const ToasterContainer = styled.div`
-  position: absolute;
-  bottom: 0;
-
   &:not([data-has-been-frozen='true']) {
     animation: ${animations('emergeSlantFromTop')};
   }
+  height: auto;
+  max-height: 100px;
+  transition: ${transition('max-height', { duration: 'm' })};
```
Comment (jean9696) [inline src/notify/ToasterList.style.ts]:
```suggestion
  transition: ${transition('all', { duration: 'm' })};
```

**Review by jean9696 — CHANGES_REQUESTED** [review]

File: src/notify/ToasterList.interface.ts (hunk 1 above)
Comment (Carolinevp) [reply src/notify/ToasterList.interface.ts]: We don't pass it anymore at [initialization](https://github.com/habx/ui-core/pull/2307/files#diff-ec290ded78d414993450a1a3fd58e953d503dffe5e6e306098a585d3c26b634eL114)

Comment (jean9696) [reply src/notify/ToasterList.interface.ts]: we should put `null` then

Comment (Carolinevp) [reply src/notify/ToasterList.interface.ts]: What does it change in the end?

**Review by jean9696 — APPROVED** [review]

**Pull request merged**